Pinta.Core/Actions/EditActions.cs

```diff
@@ -502,7 +503,21 @@ private async void HandlerPintaCoreActionsEditLoadPaletteActivated (object sende
 			return;
 
 		last_palette_dir = choice.GetParent ();
-		palette.CurrentPalette.Load (palette_formats, choice);
+
+		try {
+			palette.CurrentPalette.Load (palette_formats, choice);
+
+		} catch (PaletteLoadException ex) {
+
+			var parent = chrome.MainWindow;
+
+			await chrome.ShowUnsupportedFormatDialog (
+				parent,
+				palette_formats.Formats,
+				ex.FileName,
+				Translations.GetString ("Unsupported palette format"),
+				(ex.InnerException?.Message) ?? ex.Message);
```

Comment:
This can just be `ex.Message` now that we're directly throwing a `PaletteLoadException`?